src/TextEditor.js

```diff
@@ -7,9 +7,9 @@
 import React, {Component, PropTypes} from 'react';
 import classNames from 'classnames';
 import Immutable from 'immutable';
-import { Editor, EditorState, ContentState, convertFromHTML, RichUtils, convertToRaw, convertFromRaw, CompositeDecorator } from 'draft-js';
+import { Editor, EditorState, ContentState, RichUtils, convertToRaw, convertFromRaw, CompositeDecorator, Modifier } from 'draft-js';
 import { stateToHTML } from 'draft-js-export-html';
-
+import {convertFromHTML} from 'draft-convert';
```

Comment:
Removed draft-js-export-html module and switched to draft-convert. Apparently draft-convert doesn't need htmlOptions object to pass in (does have that flexibility though in case we need). Feel free to test out the new changes on kards ppship.